.sequelizerc

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env node
 const path = require('path')
 
+const migrationsDir =
+  process.env.IS_TESTING_DB_MIGRATION === "true" ?
```

Comment:
Should we rather have the whole path in the env variable and fall back to the default value (which is the production one)?

Reply:
Okay, I've refactor the test a bit, so this file still the same as here, but the other one, db-migration-utils.ts now does NOT have any test purpose code.

The reason to NOT mock the exec is, it is quite hard as the code also check against db, so need to mock many of those db query as well, make the test quite hard to write (and read).

Reply:
Refactor the test again to have unit test and some test on the handler itself to with mock on the exec and some db query.